src/Microsoft.Data.SqlClient.Extensions/Azure/test/Azure.Test.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'd still personally prefer it if test projects go in the root /test/ rather than under the project. But, we can argue about that later.